<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TegraK1_Performance</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="2">
    <layout class="QGridLayout" name="modifyParametersLayout"/>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout" name="controlPanelLayout"/>
   </item>
   <item row="0" column="0" rowspan="2" colspan="2">
    <layout class="QVBoxLayout" name="benchmarkLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
